demo/topdown_demo_with_mmdet.py

```diff
@@ -172,7 +175,11 @@ def main():
             pose_estimator,
             visualizer,
             show_interval=0)
-        pred_instances_list = split_instances(pred_instances)
+
+        if pred_instances is not None:
+            pred_instances_list = split_instances(pred_instances)
+        else:
+            pred_instances_list = []
```

Comment:
Would it be better if we improved the `split_instances` function to handle `None` inputs and return an empty list?

Reply:
good idea